re-using the "_" symbol

The following program `sample.go` triggers a panic:

```go
package main

var _ = true
var _ = "hello"

func main() {
    println("hello")
}
```

Expected result:
```console
$ go run ./sample.go
hello
```

Got:
```console
$ yaegi ./sample.go
./sample.go:3:5: panic
panic: reflect.Set: value of type bool is not assignable to type string [recovered]
        panic: reflect.Set: value of type bool is not assignable to type string

goroutine 1 [running]:
github.com/containous/yaegi/interp.runCfg.func1(0xc000260000, 0xc000180400)
        /home/marc/go/src/github.com/containous/yaegi/interp/run.go:114 +0x218
panic(0xc9daa0, 0xc0001504a0)
        /usr/lib/go/src/runtime/panic.go:679 +0x1b2
reflect.Value.assignTo(0xc96de0, 0x14f5d61, 0x81, 0xddc2f2, 0xb, 0xc9daa0, 0x0, 0xc0001b1c10, 0x444cc3, 0xc9daa0)
        /usr/lib/go/src/reflect/value.go:2403 +0x432
reflect.Value.Set(0xc9daa0, 0xc000150490, 0x198, 0xc96de0, 0x14f5d61, 0x81)
        /usr/lib/go/src/reflect/value.go:1537 +0xbd
github.com/containous/yaegi/interp.assign.func3(0xc000260000, 0x1546060)
        /home/marc/go/src/github.com/containous/yaegi/interp/run.go:303 +0xb6
github.com/containous/yaegi/interp.runCfg(0xc000180400, 0xc000260000)
        /home/marc/go/src/github.com/containous/yaegi/interp/run.go:120 +0x7a
github.com/containous/yaegi/interp.(*Interpreter).run(0xc000251e40, 0xc000180100, 0x0)
        /home/marc/go/src/github.com/containous/yaegi/interp/run.go:98 +0x239
github.com/containous/yaegi/interp.(*Interpreter).Eval(0xc000251e40, 0xc000268000, 0x4e, 0x24e, 0xc000268000, 0x4e, 0x0, 0x1)
        /home/marc/go/src/github.com/containous/yaegi/interp/interp.go:364 +0x3ef
main.main()
        /home/marc/go/src/github.com/containous/yaegi/cmd/yaegi/yaegi.go:82 +0x42e
```

--- a/_test/var10.go
+++ b/_test/var10.go
@@ -1,0 +1,11 @@
+package main
+
+var _ = true
+var _ = "hello"
+
+func main() {
+	println("hello")
+}
+
+// Output:
+// hello

--- a/_test/var11.go
+++ b/_test/var11.go
@@ -1,0 +1,10 @@
+package main
+
+var a, _, _, b = 1, true, "foo", 2
+
+func main() {
+	println(a, b)
+}
+
+// Output:
+// 1 2

--- a/interp/run.go
+++ b/interp/run.go
@@ -293,12 +293,17 @@ func assign(n *node) {
 	}
 
 	if n.nleft == 1 {
-		if s, d, i := svalue[0], dvalue[0], ivalue[0]; i != nil {
+		switch s, d, i := svalue[0], dvalue[0], ivalue[0]; {
+		case n.child[0].ident == "_":
+			n.exec = func(f *frame) bltn {
+				return next
+			}
+		case i != nil:
 			n.exec = func(f *frame) bltn {
 				d(f).SetMapIndex(i(f), s(f))
 				return next
 			}
-		} else {
+		default:
 			n.exec = func(f *frame) bltn {
 				d(f).Set(s(f))
 				return next
@@ -316,7 +321,6 @@ func assign(n *node) {
 			default:
 				t = typ.TypeOf()
 			}
-			//types[i] = n.child[sbase+i].typ.TypeOf()
 			types[i] = t
 		}
 
@@ -326,10 +330,16 @@ func assign(n *node) {
 		n.exec = func(f *frame) bltn {
 			t := make([]reflect.Value, len(svalue))
 			for i, s := range svalue {
+				if n.child[i].ident == "_" {
+					continue
+				}
 				t[i] = reflect.New(types[i]).Elem()
 				t[i].Set(s(f))
 			}
 			for i, d := range dvalue {
+				if n.child[i].ident == "_" {
+					continue
+				}
 				if j := ivalue[i]; j != nil {
 					d(f).SetMapIndex(j(f), t[i]) // Assign a map entry
 				} else {

